core/src/main/java/com/amplifyframework/auth/AuthException.java

```diff
@@ -61,7 +61,8 @@ public AuthException(
     public static class InvalidStateException extends AuthException {
         private static final long serialVersionUID = 1L;
         private static final String MESSAGE = "Auth state is an invalid state, cannot process the request.";
-        private static final String RECOVERY_SUGGESTION = "Please reset auth plugin or reattempt the operation later.";
+        private static final String RECOVERY_SUGGESTION = "Operation performed is not a valid " +
```

Comment:
I wanted to call this out. I made the change to match iOS and the suggestion for "Invalid State" seemed much better than our current.

Reply:
Recovery suggestion should have an action that customer can take. If we don't have any, there are other messages too. 
Here, both `MESSAGE` and `RECOVERY_SUGGESTION` convey the message of what went wrong but no suggested action.